e2e/testsuite/testsuite.go

```diff
@@ -172,9 +169,17 @@ func (s *E2ETestSuite) newInterchain(ctx context.Context, r ibc.Relayer, chains
 	for i := 0; i < len(chains)-1; i++ {
 		pathName := s.generatePathName()
 		pathNames = append(pathNames, pathName)
+
+		chain1 := chains[i]
+		chain2 := chains[i+1]
+		channelOptions := defaultChannelOpts(chains)
+		if channelOpts != nil {
+			channelOpts(&channelOptions, chain1, chain2)
```

Comment:
This way we can actually set up the channel options depending on the chains we are connecting.